### ISSUE TITLE
Improve AirPlay and lock screen integration for iOS 13

### ISSUE THREAD

**Issue by defagos** [SRGSSR/srgletterbox-apple#184]
iOS 13 introduced a few major improvements to the AirPlay experience, targeting big screen, as discussed in [WWDC 501](https://developer.apple.com/videos/play/wwdc2019/501/):

- [x] Subtitle and audio track selection.
- [x] Long-form playback. ~Nothing to be done~
- [x] Siri integration (e.g. seeks).
- [x] ~Reduce now info center refreshes (only when major changes occur, e.g. a seek, metadata change). The system interpolates in normal conditions.~ (moved to #182).

The talk also provides a few recommendations which are probably worth consideration (e.g. activate an audio session when needed).

**Comment by defagos** [comment]
I attempted implementing subtitle and audio track selection from the control center and lock screen. I sadly discovered that this is not working yet, at least not in iOS 13.2b2. It namely suffices to run the associated [official sample code](https://developer.apple.com/documentation/mediaplayer/becoming_a_now_playable_app) to see that the feature does not appear in the control center.

The code (in an unfinished state, as the selection methods still need to be implemented) is available on the `feature/language-selection` branch. We should not merge it back until we can make it work.

**Comment by defagos** [comment]
Long-form playback has been documented. Its implementation is the responsibility of the application and Letterbox itself does not need any change to support it.

Initially planned for 1.15.1, I would say this issue should be put on hold for the moment. I'll check language selection again in the future.

**Comment by defagos** [comment]
Has been delivered in 3.0.0.
